doc/setup.md

```diff
@@ -19,6 +19,8 @@ symfony new . 2.8
 composer require victoire/victoire \
     friendsofsymfony/user-bundle:~2.0@dev \
     doctrine/orm \
+    doctrine/doctrine-fixtures-bundle \
```

Comment:
Indeed, the setup specifies `new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle()`